src/OrchardCore.Modules/OrchardCore.ReverseProxy/Views/ReverseProxySettings.Edit.cshtml

```diff
@@ -33,6 +33,14 @@
     <span class="hint">@T["Current request value;"] @Context.Request.Host</span>
 </fieldset>
 
-<span class="hint">
-    Documentation available at; https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/proxy-load-balancer?view=aspnetcore-2.1
-</span>
\ No newline at end of file
+<div class="row">
+    <span class="hint">
+        @T["The current tennant will be reloaded when settings are updated."]
+    </span>
+</div>
+
```

Comment:
The warning is normally at the top of the page.

https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore.Modules/OrchardCore.Https/Views/HttpsSettings.Edit.cshtml
for example.

Plus then the text is the same as elsewhere when people are doing translations.
```
<p class="alert alert-warning">@T["The current tenant will be reloaded when the settings are saved."]</p>
```